main/options.c

```diff
@@ -439,6 +440,10 @@ static optionDescription LongOptionDescription [] = {
  {0,"  --_interactive"},
  {0,"       Enter interactive mode (json over stdio)."},
 #endif
+#ifdef HAVE_SECCOMP
+ {0,"  --_secure"},
+ {0,"       Enter secure interactive mode (json over stdio, input over stdin)."},
+#endif
```

Comment:
This list is in alphabetical order.
So this block should be moved between `--_list-roles` and `--_xformat`.